pkg/annotation/annotation.go

```diff
@@ -10,4 +10,8 @@ const (
 	// CordonUntilDate is the name of the annotation that indicates
 	// the expiration date of rule of this cordon.
 	CordonUntilDate = "chart-operator.giantswarm.io/cordon-until"
+
+	// DeleteCustomResourceOnly is the name of the annotation that indicates
+	// the custom resource should be deleted without deleting the Helm release.
+	DeleteCustomResourceOnly = "chart-operator.giantswarm.io/delete-custom-resource-only"
```

Comment:
Is the value ignored?

Reply:
I think we don't need to care of value in here.

Reply:
Yes it's just the presence of the annotation that matters.

Reply:
In the key helper it now checks the value is true.

Reply:
I missed using the key func in the current state but this is now fixed. It does now check the value is true and I agree that is better.